Meta/gn/secondary/Meta/Lagom/Contrib/MacPDF/compile_xib_resources.gni

```diff
@@ -0,0 +1,22 @@
+template("compile_xib_resources") {
+  action_foreach(target_name) {
+    forward_variables_from(invoker, [ "sources" ])
+
+    # FIXME: Move this helper skript to a common location?
+    # FIXME: Be clever and create a custom script that will write a depfile
+    script = "//Meta/gn/secondary/Ladybird/invoke_process_with_args.py"
```

Comment:
🤨

Maybe that script should move to Meta/gn directly? Or Meta/gn/scripts or something.

Also, if we only need this template in one place, we could make it not a template. Or if we do want to make it a template for future uses in other places, maybe it should also go into some neutral place below Meta/gn?